keda-operator crashs with SIGSEGV for kafka scaler

## Expected Behavior
If the offset lag cannot be polled return 0 and do not crash with SIGSEGV.

## Actual Behavior
keda-operator crashs with SIGSEGV:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x13c43ad]
goroutine 1544 [running]:
github.com/kedacore/keda/pkg/scalers.(*kafkaScaler).getLagForPartition(0xc0008f03c0, 0xc000000017, 0xc000978de0, 0x32)
	keda/pkg/scalers/kafka_scaler.go:281 +0x5d
github.com/kedacore/keda/pkg/scalers.(*kafkaScaler).IsActive(0xc0008f03c0, 0x21c2060, 0xc0017d0200, 0x1, 0x1, 0xc0016d2900)
	keda/pkg/scalers/kafka_scaler.go:168 +0xe0
github.com/kedacore/keda/pkg/handler.(*ScaleHandler).handleScaleDeployment(0xc0008aa630, 0x21c2060, 0xc0017d0200, 0xc000b96340)
	keda/pkg/handler/scale_loop.go:119 +0x11a
github.com/kedacore/keda/pkg/handler.(*ScaleHandler).handleScale(0xc0008aa630, 0x21c2060, 0xc0017d0200, 0xc000b96340)
	keda/pkg/handler/scale_loop.go:45 +0x6b
github.com/kedacore/keda/pkg/handler.(*ScaleHandler).HandleScaleLoop(0xc0008aa630, 0x21c2060, 0xc0017d0200, 0xc000b96340)
	keda/pkg/handler/scale_loop.go:14 +0x1d2
created by github.com/kedacore/keda/pkg/controller/scaledobject.(*ReconcileScaledObject).startScaleLoop
	keda/pkg/controller/scaledobject/scaledobject_controller.go:333 +0x35f
```

## Steps to Reproduce the Problem

This problem occurs only when a kafka topic partition is not clean and ready to use. I have seen that when a broker as partition leader restarts.

## Specifications

  - **KEDA Version:** 1.2.0
  - **Platform & Version:** GKE COS
  - **Kubernetes Version:** v1.14.10-gke.17/v1.15.9-gke.8
  - **Scaler(s):** kafka

--- a/pkg/scalers/kafka_scaler.go
+++ b/pkg/scalers/kafka_scaler.go
@@ -288,6 +288,10 @@ func (s *kafkaScaler) getOffsets(partitions []int32) (*sarama.OffsetFetchRespons
 
 func (s *kafkaScaler) getLagForPartition(partition int32, offsets *sarama.OffsetFetchResponse) int64 {
 	block := offsets.GetBlock(s.metadata.topic, partition)
+	if block == nil {
+		kafkaLog.Error(fmt.Errorf("error finding offset block for topic %s and partition %d", s.metadata.topic, partition), "")
+		return 0
+	}
 	consumerOffset := block.Offset
 	latestOffset, err := s.client.GetOffset(s.metadata.topic, partition, sarama.OffsetNewest)
 	if err != nil {

